### PR TITLE
Remove duplicates

### PR DESCRIPTION
**Pull request by topecongiro**
This PR removes duplicated definitions of `type_annotation_separator` and `type_bound_colon`.
Also fixes a typo in the document of `src/lists.rs`.

### CODE REVIEW

File: src/utils.rs [hunk 1]
```diff
@@ -326,6 +326,16 @@ pub fn binary_search<C, T>(mut lo: usize, mut hi: usize, callback: C) -> Option<
     None
 }
 
+#[inline]
+pub fn place_spaces(before: bool, after: bool) -> &'static str {
```
Comment (nrc) [inline src/utils.rs]: Perhaps `colon_spaces` would be a better name?

**Comment by nrc** [comment]
Tests are also failing. You can run `cargo test` locally to check tests. I think you only need to reformat your code.

**Comment by topecongiro** [comment]
@nrc Thanks! I was embarrassingly running `cargo test` on master branch...
Fixed & Rebased.

**Comment by nrc** [comment]
Thanks!

**Pull request merged**